_config.yml

```diff
@@ -1,13 +1,13 @@
 # Site settings
 title: Sensu Plugins
 
-support_email: support@sensuapp.org
-irc: irc.freenode#sensu
+support_email: community@sensu.io
```

Comment:
should the support email really be community? I feel like they should probably be kept seperate for at least SLA tracking purposes. Totally not my call nor do I really care beyond trying to help thinking about it.

Reply:
It's a tough one. Team Sensu support should not have a direct link from this community site - it's out of scope of their work. I think of support on this page as a request to the community team at the company. The only thing I'm sure of is it shouldn't be the SLA'ed email of our customer support.

Reply:
Leaving as is so Community team can monitor community emails and not expect our customer support team to receive community requests.